### PR TITLE
docs:Add tidb_opt_force_inline_cte system vars usage description in docs.

### PR DESCRIPTION
**Pull request by elsa0520**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/tidb#36514

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
 - 默认值：`OFF`
 - 这个变量用来设置是否允许 `INSERT`、`REPLACE` 和 `UPDATE` 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
 
+### `tidb_opt_inline_cte` <span class="version-mark">从 v6.4 版本开始引入</span>
```
Comment (AilinKid) [inline system-variables.md:1835]: v6.3.0 still unreleased？

Comment (elsa0520) [reply system-variables.md:1835]: What version of the master code do we have now? 
I don't know if my code can be merged before 6.3 is released, so here I will fill in 6.4 firstly

Comment (shichun-0415) [reply system-variables.md:1835]: Do we have an FD jira issue in 6.3? If not, we cannot release it in 6.3.

Comment (elsa0520) [reply system-variables.md]: I'm afraid not, there is only one issue link at the moment.

File: system-variables.md [hunk 2]
```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
 - 默认值：`OFF`
 - 这个变量用来设置是否允许 `INSERT`、`REPLACE` 和 `UPDATE` 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
 
+### `tidb_opt_force_inline_cte` <span class="version-mark">从 v6.4 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔
+- 默认值：`OFF`
+- 这个变量主要用来控制是否强制开启 inline CTE。默认值为 OFF，则默认不强制开启 inline CTE。（但依旧可以通过 MERGE（）hint 来开启个别 CTE 的 inline）。如果设置为 ON，则当前 session 中所有查询的 CTE 都会是 inline。
```
Comment (fixdb) [inline system-variables.md:1841]: recursive CTE除外

File: system-variables.md (hunk 1 above)
Comment (fixdb) [reply system-variables.md]: @shichun-0415 There is a FD: FD-999

Comment (fixdb) [reply system-variables.md]: target version: 6.3

File: system-variables.md (hunk 2 above)
Comment (shichun-0415) [inline system-variables.md:1841]:
```suggestion
- 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE，注意，此时依旧可以通过 `MERGE（）hint` 来开启个别 CTE 的 inline。如果设置为 `ON`，则当前 session 中所有查询的 CTE 都会是 inline。
```

File: system-variables.md (hunk 1 above)
Comment (elsa0520) [reply system-variables.md]: done

File: system-variables.md [hunk 3]
```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
 - 默认值：`OFF`
 - 这个变量用来设置是否允许 `INSERT`、`REPLACE` 和 `UPDATE` 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
 
+### `tidb_opt_force_inline_cte` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔
+- 默认值：`OFF`
+- 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE，注意，此时依旧可以通过 `MERGE（）hint` 来开启个别 CTE 的 inline。如果设置为 `ON`，则当前 session 中所有查询的 CTE 都会是 inline ( recursive CTE 除外)。
```
Comment (shichun-0415) [inline system-variables.md:1841]:
```suggestion
- 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE，注意，此时依旧可以通过 `MERGE（）hint` 来开启个别 CTE 的 inline。如果设置为 `ON`，则当前 session 中所有查询的 CTE（recursive CTE 除外）都会是 inline 。
```

Comment (elsa0520) [reply system-variables.md]: fixed

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 4]
```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
 - 默认值：`OFF`
 - 这个变量用来设置是否允许 `INSERT`、`REPLACE` 和 `UPDATE` 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
 
+### `tidb_opt_force_inline_cte` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔
```
Comment (TomShawn) [inline system-variables.md:1839]:
```suggestion
- 类型：布尔型
```

File: system-variables.md [hunk 5]
```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
...
+- 默认值：`OFF`
+- 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE，注意，此时依旧可以通过 `MERGE（）hint` 来开启个别 CTE 的 inline。如果设置为 `ON`，则当前 session 中所有查询的 CTE (recursive CTE 除外) 都会是 inline 。
```
Comment (TomShawn) [inline system-variables.md:1841]:
```suggestion
- 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE。注意，此时依旧可以通过 `MERGE()` hint 来开启个别 CTE 的 inline。如果设置为 `ON`，则当前 session 中所有查询的 CTE（递归 CTE 除外）都会是 inline。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: system-variables.md [hunk 6]
```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
 - 默认值：`OFF`
 - 这个变量用来设置是否允许 `INSERT`、`REPLACE` 和 `UPDATE` 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
 
+### `tidb_opt_force_inline_cte` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`OFF`
+- 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE。注意，此时依旧可以通过 `MERGE()` hint 来开启个别 CTE 的 inline。如果设置为 `ON`，则当前 session 中所有查询的 CTE（递归 CTE 除外）都会是 inline。
```
Comment (fixdb) [inline system-variables.md:1841]: 都会是 inline 
-> 
都会 inline

Comment (fixdb) [inline system-variables.md:1841]: 这个变量用来控制是否强制开启 inline CTE。
->
这个变量用来控制是否强制 inline CTE。

Comment (fixdb) [reply system-variables.md:1841]: - 这个变量用来控制是否强制开启 inline CTE。默认值为 `OFF`，即默认不强制开启 inline CTE

->

- 这个变量用来控制是否强制 inline CTE。默认值为 `OFF`，即默认不强制 inline CTE

**Review by fixdb — APPROVED** [review]

**Pull request merged**